web/auth/auth.go

```diff
@@ -1166,6 +1183,7 @@ func Routes(router *echo.Group) {
 	router.POST("/passphrase_reset", passphraseReset, noCSRF)
 	router.GET("/passphrase_renew", passphraseRenewForm, noCSRF)
 	router.POST("/passphrase_renew", passphraseRenew, noCSRF)
+	router.GET("/passphrase", passphraseForm, noCSRF)
```

Comment:
You know the rules: if you add a route, you need to add documentation for it. And for things in `/auth`, you need to have a very good reason to not add a unit test.